IAM/Frequently_asked_questions.mediawiki

```diff
@@ -211,11 +211,57 @@ If you own a volunteer LDAP account, then you can upgrade your mozillians profil
 
 ==== '''Q''': ''What issues I might encounter by upgrading to Firefox Accounts?'' ====
 
-There are some known issues with using Firefox Accounts in Mozilla IAM: 
+There are some known issues with using Firefox Accounts in Mozilla IAM:
 
 1.  Login with Firefox Accounts is unavailable from inside some Android applications, including IRCCloud and Slack. This is due to lack of localStorage support in some Android WebViews.
 2.  Firefox Accounts can only be used in Mozilla IAM with 2FA enabled. Note that once you choose to use Firefox Accounts, it is required to set up 2FA, to avoid being locked out of your account.
 
+==== '''Q''': ''Why do the email login links expire after 15 minutes?'' ====
+
+When you login using an email link, that link is valid for 15 minutes from when you request it.
+This expiration window of 15 minutes is driven both by security and a desire for
+a positive user experience. The link is short lived so that there is a limited window
+of time during which an attacker could use the link if they were able to get access
+to it. This is especially important due to the inherently insecure nature of email
+transmission. The user experience of having to wait for over 15 minutes to log into
```

Comment:
```suggestion
transmission. 
```

Reply:
From this explanation, it's unclear to me why the 15min expiry interval would contribute a waiting period to the UX. Perhaps this is why @hmitsch is looking to remove? @gene1wood is there further clarity you can provide so the tieback you're looking to make isn't lost?

Reply:
@gene1wood I totally see what you mean and agree to the use case and reasoning you lay out. I’d still drop that last sentence mentioning user experience because I think it is confusing. If you have a strong opinion that the crux of the FAQ entry is this last sentence, let’s keep it. I do not want to block publishing this.

Reply:
Thanks @gene1wood this is a very concise summary and I'm sure we're aligned now.

I agree w/ @hmitsch that we should drop this line from this explanation as it distracts from the point you make in the following paragraph about timing and the intro of Greylisting. That paragraph does a more direct job of stating that delays happen, it's not something we have control over and it's not something we're willing to compromise security for. That in itself is the good UX. :)